testing/database.rst

```diff
@@ -113,44 +113,32 @@ not to overwrite data you entered when developing the application and also
 to be able to clear the database before every test.
 
 To do this, you can specify a database configuration which overwrites the default
-configuration:
+configuration just in the ``test`` environment:
 
 .. configuration-block::
 
     .. code-block:: yaml
 
-        # app/config/config_test.yml
+        # config/packages/test/doctrine.yaml
         doctrine:
             # ...
             dbal:
-                host:     localhost
-                dbname:   testdb
-                user:     testdb
-                password: testdb
+                url: 'mysql://USERNAME:PASSWORD@127.0.0.1/DB_NAME?charset=utf8mb4&serverVersion=5.7'
```

Comment:
As Doctrine configuration has defined an environment variable (`'%env(resolve:DATABASE_URL)%'`) for all env, I guess we should do the changes in `phpunit.xml.dist` file, adding `<env name="DATABASE_URL" value="..."/>`? then this whole configuration will be gone.

Reply:
Much better. Thanks!